start.py

```diff
@@ -55,34 +56,16 @@ def main(args):
                        for hostname in args.secondary_nodes]
 
     cluster = Cluster(primary_node, *secondary_nodes)
+    # MapR 6.0.0-rc1 uses CentOS 7 which  needs following settings.
+    if args.mapr_version == '6.0.0-rc1':
+        for node in cluster.nodes:
+            node.volumes.append({'/sys/fs/cgroup': '/sys/fs/cgroup'})
+            with tempfile.TemporaryDirectory() as tmpdirname:
```

Comment:
Don't use the context manager to create a temporary directory; it may be removed when garbage collection happens in Python. Just use `tempfile.mkdtemp` to create a directory.